src/pubsub/floodsub.ts

```diff
@@ -12,8 +13,9 @@ export function floodsubTests (factory: DaemonFactory) {
     for (const typeB of nodeTypes) {
       runFloodsubTests(
         factory,
-        { type: typeA, pubsub: true, pubsubRouter: 'floodsub' },
-        { type: typeB, pubsub: true, pubsubRouter: 'floodsub' }
+        // RSA key ensures the `key` field is set in the generated signed message
```

Comment:
Since https://github.com/libp2p/js-libp2p-pubsub/pull/118 has been merged can all the pubsub related changes can be reverted, or are they still necessary?

Reply:
It seems like tests are still failing with: 
```
  libp2p:pubsub starting +8ms
  libp2p:pubsub started +5ms
  libp2p:pubsub connected 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +60ms
  libp2p:pubsub new peer 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +48ms
  libp2p:pubsub rpc from 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +1ms
  libp2p:pubsub subscribe to topic: test-topic +3ms
  libp2p:pubsub rpc from 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +802ms
  libp2p:pubsub messages from 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +1ms
  libp2p:pubsub:error CodeError: Need key when signature policy is StrictSign but it was missing
  libp2p:pubsub:error     at EventTarget.getMsgId (file:///home/ckousik/projects/js-libp2p/node_modules/@libp2p/pubsub/src/index.ts:458:17)
  libp2p:pubsub:error     at EventTarget.processMessage (file:///home/ckousik/projects/js-libp2p/node_modules/@libp2p/floodsub/src/index.ts:76:31)
  libp2p:pubsub:error     at file:///home/ckousik/projects/js-libp2p/node_modules/@libp2p/pubsub/src/index.ts:376:22
  libp2p:pubsub:error     at processTicksAndRejections (node:internal/process/task_queues:96:5)
  libp2p:pubsub:error     at file:///home/ckousik/projects/js-libp2p/node_modules/p-queue/dist/index.js:118:36 +0ms
^C  libp2p:pubsub stopping +2s
  libp2p:pubsub delete peer 12D3KooWBaiBJ1ngvSyYJYYnKuEQNWx2j4b3s2z8c3z5shGLRM8U +0ms
  libp2p:pubsub stopped +0ms
```

Could we merge this and open an issue for updating libp2p/pubsub and reverting this change?